doc/mkdocs/docs/features/arbitrary_types.md

```diff
@@ -85,37 +85,39 @@ Some important things:
 
 If you just want to serialize/deserialize some structs, the `to_json`/`from_json` functions can be a lot of boilerplate.
 
-There are two macros to make your life easier as long as you (1) want to use a JSON object as serialization and (2) want to use the member variable names as object keys in that object:
+There are four macros to make your life easier as long as you (1) want to use a JSON object as serialization and (2) want to use the member variable names as object keys in that object:
 
 - `NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE(name, member1, member2, ...)` is to be defined inside of the namespace of the class/struct to create code for.
 - `NLOHMANN_DEFINE_TYPE_INTRUSIVE(name, member1, member2, ...)` is to be defined inside of the class/struct to create code for. This macro can also access private members.
+- `NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE_WITH_DEFAULT(name, member1, member2, ...)` is to be defined inside of the namespace of the class/struct to create code for.
```

Comment:
It would be good to add a comment about the difference to the previous macros.

Furthermore, the order should be.

- NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE
- NLOHMANN_DEFINE_TYPE_NON_INTRUSIVE_WITH_DEFAULT
- NLOHMANN_DEFINE_TYPE_INTRUSIVE
- NLOHMANN_DEFINE_TYPE_INTRUSIVE_WITH_DEFAULT